<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetTapeCounter</class>
 <widget class="QWidget" name="WidgetTapeCounter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>50</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="ziffer2">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ziffer1">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ziffer0">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
